### PR TITLE
Disable include metadata tests if external server used in server integration tests

### PR DESCRIPTION
**Pull request by brfrn169**
Please take a look!

### CODE REVIEW

**Review by Torch3333 — APPROVED** [review]
LGTM, thank you!

File: server/src/integration-test/java/com/scalar/db/server/ConsensusCommitAdminIntegrationTestWithDistributedTransactionAdminService.java [hunk 1]
```diff
@@ -41,6 +44,8 @@ protected void initialize(String testName) throws IOException {
       properties.setProperty(ServerConfig.PORT, PORT_FOR_SERVER_WITH_INCLUDE_METADATA_ENABLED);
       serverWithIncludeMetadataEnabled = new ScalarDbServer(properties);
       serverWithIncludeMetadataEnabled.start();
+    } else {
+      isExternalServerUsed = true;
```
Comment (feeblefakie) [inline server/src/integration-test/java/com/scalar/db/server/ConsensusCommitAdminIntegrationTestWithDistributedTransactionAdminService.java:48]: I'm not really sure what the external server means here. Is it true when the Scalar DB server is used in the tests?

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!
The code looks good, but I left one question for clarification.

File: server/src/integration-test/java/com/scalar/db/server/ConsensusCommitAdminIntegrationTestWithDistributedTransactionAdminService.java (hunk 1 above)
Comment (brfrn169) [reply server/src/integration-test/java/com/scalar/db/server/ConsensusCommitAdminIntegrationTestWithDistributedTransactionAdminService.java:48]: By default, we start the Scalar DB Server in the integration test JVM internally and run the tests on it. But when we specify the JVM option `scalardb.server.external_server_used=true`, we don't start the Scalar DB Server in the integration test JVM and run the tests on the external server.

**Pull request merged**